src/number_theory/bernoulli.lean

```diff
@@ -7,6 +7,7 @@ import data.rat
 import data.fintype.card
 import algebra.big_operators.nat_antidiagonal
 import ring_theory.power_series.well_known
+import algebra.group_with_zero.basic
```

Comment:
Any reason for this import?

Reply:
Yes, but I think I stopped using it at some point. I have removed it, thank you!